module2/src/com/epam/arraysofarrays/MatrixInitializer.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.arraysofarrays;
+
+public class MatrixInitializer {
+    public static int[][] buildRandomSquareMatrixWithPositiveElements(int n) {
+        int[][] matrix = new int[n][n];
+        for (int i = 0; i < n; i++) {
+            for (int j = 0; j < n; j++) {
+                matrix[i][j] = (int) (Math.random() * 100);
+            }
+        }
+        return matrix;
+    }
+    public static int [][] buildRandomSquareMatrixWithPositiveAndNegativeElements(int n) {
+        int[][] matrix = new int[n][n];
+        for (int i = 0; i < n; i++) {
+            for (int j = 0; j < n; j++) {
+                matrix[i][j] = ((int) (Math.random() * 100) - 50);
+            }
+        }
+        return matrix;
+    }
+    public static int[][] buildRandomMatrix10x20 (int rows, int columns){
```

Comment:
You can replace all or at least some of these methods into one if you make them more general with params of min and max borders of randomization